### PR TITLE
[CI:DOCS] Add troubleshooting advice about the --userns option.

### PR DESCRIPTION
**Pull request by rhatdan**
Also a link to the troubleshooting guide into the issue template.

Replaces: https://github.com/containers/podman/pull/9770

Signed-off-by: Josh Berkus <josh@agliodbs.com>
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: troubleshooting.md [hunk 1]
```diff
@@ -47,6 +47,14 @@ will disable SELinux separation for the container.
...
 
+In cases where the container image runs as a specific, non-root user, though, the
+solution is to fix the user namespace.  This would include container images such as
+the Jupyter Notebook image (which runs as "jovyan") and the Postgres image (which runs
+as "postgres").  In either case, use the `--userns` switch to map user namespaces,
+most of the time by using keep_id option.
+
+$ podman run -v "$PWD":/home/jovyan/work --userns=keep_id jupyter/scipy-notebook
+
 ---
```
Comment (TomSweeneyRedHat) [inline troubleshooting.md:58]: Do we need to also call `system migrate` as we show in #10 in this doc?

Comment (rhatdan) [reply troubleshooting.md:58]: No migrate is only needed if you have to modify /etc/subuid and /etc/subgid.

Comment (TomSweeneyRedHat) [reply troubleshooting.md:58]: I was asking as I assumed "fix the usernamespace" meant jiggling the settings in /etc/subuid and /etc/subgid.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**